app/tests/spec/lib/experiments/grouping-rules/email-first.js

```diff
@@ -0,0 +1,70 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define(function (require, exports, module) {
+  'use strict';
+
+  const { assert } = require('chai');
+  const Experiment = require('lib/experiments/grouping-rules/email-first');
+  const sinon = require('sinon');
+
+  describe('lib/experiments/grouping-rules/email-first', () => {
+    let experiment;
+    let experimentGroupingRules;
+
+    before(() => {
+      experiment = new Experiment();
+      experimentGroupingRules = {
+        choose (experimentName) {
+          return experimentName === 'isSampledUser';
```

Comment:
Yup, the only time `experimentGroupingRules.choose` is called within the experiment is to see if the user is in the DataDog metrics. I can add a note about that.